lib/logging_http_client.rb

```diff
@@ -0,0 +1,33 @@
+require 'time'
```

Comment:
Should this class be in the main `lib` dir? It is after all an example --- it says so in the code comments.